ocaml/xapi/vm_platform.ml

```diff
@@ -196,21 +196,18 @@ let sanity_check ~platformdata ?firmware ~vcpu_max ~vcpu_at_startup ~domain_type
         if cps < 1 || vcpus mod cps <> 0 then
           raise
             (Api_errors.Server_error
-               ( Api_errors.invalid_value
+               ( Api_errors.vcpu_max_not_cores_per_socket_multiple
                , [
                    Printf.sprintf "platform:cores-per-socket (value %d)" cps
-                 ; Printf.sprintf
-                     "VCPUs_max (value %d) must be a multiple of \
-                      cores-per-socket"
-                     vcpus
+                 ; Printf.sprintf "%d" vcpus
                  ] ))
       with Failure msg ->
         raise
           (Api_errors.Server_error
-             ( Api_errors.invalid_value
```

Comment:
This should still be `Api_errors.invalid_value`: it's a string that cannot be transformed into a string. i don't think `vcpus` should be displayed at all

Reply:
Thank you! :)